config/v1/types_cluster_version.go

```diff
@@ -190,6 +201,10 @@ type UpdateHistory struct {
 	// +kubebuilder:validation:Required
 	// +required
 	Verified bool `json:"verified"`
+
+	// overrides records update guards which were overriden to initiate the update.
```

Comment:
free-form, no format, never cleared, empty means "no guards were overridden".  I dunno how much of that is worth including in the godocs.  "The godocs do not describe a format" sounds functionally indistinguishable to "This field is a free-form string" to me.  And existing history entries (e.g. `verified`) are also effectively write-once today, but don't call that out explicitly.  And many optional-string fields are "empty means there is nothing like this" (in this case "there were no overrides"), because otherwise we'd need a `*string`, and this is a non-pointer `string`.

Reply:
I can't figure out how I would use this field. What is an update guard?

Reply:
humans read this field.  Machines possibly switch on "there is content".  Those are the only two consumers.

Update guards are "anything the CVO was unhappy about".  So "you asked me to do this even though there was a targeted edge blocking risk that applied to this cluster" would go in here.  As would "I couldn't find a signature, but you forced me to accept the update anyway" and "I was Upgradeable=False with $REASON, but you forced me to accept this minor bump anyway", etc., etc.  This is unstructured space for the CVO to complain about the things it is doing against its better judgement because the admin asked nicely.

Reply:
I've pushed 66adeda8 -> bc1d2cdfbf6 to pivot this property to `acceptedRisks`.  New docstring:

> acceptedRisks records risks which were accepted to initiate the update. For example, it may menition an Upgradeable=False or missing signature that was overriden via desiredUpdate.force, or an update that was initiated despite not being in the availableUpdates set of recommended update targets.

Acceptable?